source/configure/configuration-settings.rst

```diff
@@ -223,771 +223,81 @@ See the :doc:`Elasticsearch configuration settings </configure/elasticsearch-con
-
-**False**: All communication using gossip protocol remains unencrypted.
-
-The encryption uses AES-256 by default, and it is not kept configurable by design. However, you can manually set the ``ClusterEncryptionKey`` row value in the Systems table. A key is a byte array converted to base64. It should be either 16, 24, or 32 bytes to select AES-128, AES-192, or AES-256.
-
-+--------------------------------------------------------------------------------------------------------------------------------+
-| This feature's ``config.json`` setting is ``"EnableExperimentalGossipEncryption": false`` with options ``true`` and ``false``. |
-+--------------------------------------------------------------------------------------------------------------------------------+    
-    
-Enable Gossip Compression
-^^^^^^^^^^^^^^^^^^^^^^^^^
-
-|enterprise| |self-hosted|
-
-*Available in legacy Enterprise Edition E20*
-
-**True**: All communication through the cluster uses gossip compression. This is set to ``true`` by default to maintain compatibility with older servers.
-
-**False**: All communication using the gossip protocol remains uncompressed. Once all servers in a cluster are upgraded to Mattermost v5.33 or later, we recommend that you disable this configuration setting for better performance.
-
-+--------------------------------------------------------------------------------------------------------------------------------+
-| This feature's ``config.json`` setting is ``"EnableGossipCompression": true`` with options ``true`` and ``false``.             |
-+--------------------------------------------------------------------------------------------------------------------------------+    
-
-Gossip Port
-^^^^^^^^^^^
-
-|enterprise| |self-hosted|
-
-*Available in legacy Enterprise Edition E20*
-
-The port used for the gossip protocol. Both UDP and TCP should be allowed on this port.
-
-+-------------------------------------------------------------------------------------------+
-| This feature's ``config.json`` setting is ``"GossipPort": 8074`` with numerical input.    |
-+-------------------------------------------------------------------------------------------+
-
-Streaming Port
-^^^^^^^^^^^^^^
-
-|enterprise| |self-hosted|
-
-*Available in legacy Enterprise Edition E20*
-
-The port used for streaming data between servers.
+See the :doc:`high availability configuration settings </configure/high-availability-configuration-settings>` documentation for details on the following configuration settings:
 
-+----------------------------------------------------------------------------------------------+
-| This feature's ``config.json`` setting is ``"StreamingPort": 8075`` with numerical input.    |
-+----------------------------------------------------------------------------------------------+
+- `Enable high availability mode <https://docs.mattermost.com/configure/push-notification-server-configuration-settings.html#enable-high-availability-mode>`__
```

Comment:
```suggestion
- `Enable High Availability mode <https://docs.mattermost.com/configure/push-notification-server-configuration-settings.html#enable-high-availability-mode>`__
```

Reply:
Leaving "high availability" as-is with no capitalization.